plugins/inputs/opcua/opcua_util.go

```diff
@@ -135,6 +135,7 @@ func pemBlockForKey(priv interface{}) *pem.Block {
 	case *ecdsa.PrivateKey:
 		b, err := x509.MarshalECPrivateKey(k)
 		if err != nil {
+			//nolint:errcheck,revive
 			fmt.Fprintf(os.Stderr, "Unable to marshal ECDSA private key: %v", err)
 			os.Exit(2)
```

Comment:
Why just don't return new error here and use the caller to fail?

Reply:
Will do.